lib/dentaku/ast/arithmetic.rb

```diff
@@ -28,30 +28,18 @@ class Addition < Arithmetic
       def operator
         :+
       end
-
-      def self.precedence
```

Comment:
did we stop needing this after the parser rewrite?

Reply:
Yep! Precedence is now hardcoded in the parser, which is honestly where it should live.